themes/default/content/what-is/what-is-platform-engineering.md

```diff
@@ -2,42 +2,53 @@
 title: What is Platform Engineering?
 allow_long_title: true
 meta_desc: |
-    Understand what platform engineering is, along with the main benefits and importance
-    for modern application development.
-
+    Understand what platform engineering is, along with the main benefits and importance for modern application development.
 type: what-is
 page_title: What is Platform Engineering?
-
 ---
 
+Platform engineering is a set of modern engineering practices that take a holistic approach to managing the entire software development lifecycle, encompassing infrastructure, tools, and processes. The aim of platform engineering is to provide a scalable and secure platform that supports the development, deployment, and operation of software applications in a standardized and efficient way.
+
 ## What is platform engineering?
 
-Platform engineering is a set of modern engineering practices where infrastructure teams build shared tools and services to help development teams develop, deploy, and operate cloud infrastructure on a self-service basis. This includes cloud infrastructure, container orchestration platforms, databases, networking, monitoring, code repositories, and deployment pipelines.
+In a platform engineering approach, one or more teams---often referred to as the platform engineering team or the platform team---build a comprehensive set of shared tools and services (aka "the platform") to help development teams develop, deploy, and operate cloud infrastructure on a self-service basis. This includes cloud infrastructure, container orchestration platforms, databases, networking, monitoring, code repositories, and deployment pipelines.
 
-Platform teams use a customer-driven mindset where they treat the application developers that they serve as customers that must be understood and won over through products that solve their problems. The products these teams offer are infrastructure tools and building blocks that development teams use to provision and manage standardized infrastructure for their applications and services. Typically, these tools have built-in guardrails that enforce best practices and security standards without impeding developers’ agility and workflow.
+Platform teams typically adopt a customer-driven mindset where they treat the application developers that they serve as customers that must be understood and won over through products that solve their problems. The products these teams offer are infrastructure tools and building blocks that development teams use to provision and manage standardized infrastructure for their applications and services. Typically, these tools have built-in guardrails that enforce best practices and security standards without impeding developers' agility and workflow. This is all done in service of increasing the speed of delivery for the company's products.
```

Comment:
Should we make this a stronger opinion that they "must adopt" vs typically adopt

Reply:
I've gone with slightly stronger wording here; good suggestion.